packages/shared/src/components/squads/common/SquadPrivacyState.tsx

```diff
@@ -27,15 +28,15 @@ export function SquadPrivacyState({
   return (
     <Button
       icon={props.icon}
-      size={ButtonSize.Small}
+      size={ButtonSize.XSmall}
       variant={isFeatured ? ButtonVariant.Secondary : ButtonVariant.Float}
       className={classNames(
         'pointer-events-none',
         isFeatured &&
           'relative border-overlay-primary-cabbage bg-overlay-tertiary-cabbage',
       )}
     >
-      {props.copy} Squad
```

Comment:
Was the removal, intended here?

Reply:
Yes, in design `Squad` is not mentioned anymore